src/transaction/boot_cl.c

```diff
@@ -5456,81 +6035,57 @@ boot_define_view_partition (void)
   // *INDENT-OFF*
   sprintf (stmt,
 	"SELECT "
-	  "[pp].[super_class_name] AS [class_name], "
-	  "[pp].[super_owner_name] AS [owner_name], "
+	  "[s].[class_name] AS [class_name], "
+	  "CAST ([s].[owner].[name] AS VARCHAR(255)) AS [owner_name], " /* string -> varchar(255) */
 	  "[p].[pname] AS [partition_name], "
-	  "CONCAT ([pp].[super_class_name], '__p__', [p].[pname]) AS [partition_class_name], "
-	  "CASE "
-	      "WHEN [p].[ptype] = 0 THEN 'HASH' "
-	      "WHEN [p].[ptype] = 1 THEN 'RANGE' "
-	      "ELSE 'LIST' "
-	      "END AS [partition_type], "
-	   "TRIM (SUBSTRING ([pi].[pexpr] FROM 8 FOR (POSITION (' FROM ' IN [pi].[pexpr]) - 8))) AS [partition_expr], "
-	   "[p].[pvalues] AS [partition_values], "
-	   "[p].[comment] AS [comment] "
+	  "CONCAT ([s].[class_name], '__p__', [p].[pname]) AS [partition_class_name], " /* It can exceed varchar(255). */
+	  "CASE [p].[ptype] WHEN 0 THEN 'HASH' WHEN 1 THEN 'RANGE' ELSE 'LIST' END AS [partition_type], "
```

Comment:
it looks need to re-arge for CASE caluse.

Reply:
If there are not many branching values and does not exceed 120 bytes, it is written in one line.
If there is no other opinion, I will keep it.